Article/Article.js

```diff
@@ -22,8 +40,45 @@ class Article {
 
 - Select all classes named ".article" and assign that value to the articles variable.  
 
+
 - With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the Article class.
 
 */
 
-let articles;
\ No newline at end of file
+const mainArticle = document.querySelector('.articles');
+const submitBtn = document.querySelector('.submitArticle');
+
+submitBtn.addEventListener('click', () => {
+var div = document.createElement('div');
+var header = document.createElement('h2');
+var para = document.createElement('p');
+var closeSpan = document.createElement('span');
+var openSpan = document.createElement('span');
+
+
+openSpan.classList.add('expandButton');
+openSpan.textContent = 'Expand';
+closeSpan.classList.add('readButton');
+closeSpan.textContent = 'Close';
+headerText = document.querySelector('.newArticleTitle');
+header.textContent = headerText.value;
+paraText = document.querySelector('.newArticle');
+para.textContent = paraText.value;
+div.classList.add('article');
+
+div.appendChild(header);
+div.appendChild(para);
+div.appendChild(closeSpan);
+div.appendChild(openSpan);
+
+mainArticle.appendChild(div);
+div = new Article(div);
+console.log('submit');
+modal.style.display = "none";
+event.preventDefault();
```

Comment:
what event are you trying to prevent Default behavior of?  preventDefault behavior would typically be used inside of eventListeners that are firing in response to events.

Reply:
I was preventing it from reloading the page so I could make the data stay.